samples/client/petstore/python/test/test_format_test.py

```diff
@@ -36,19 +36,18 @@ def make_instance(self, include_optional):
         # model = petstore_api.models.format_test.FormatTest()  # noqa: E501
         if include_optional :
             return FormatTest(
-                integer = 1E+1, 
-                int32 = 2E+1, 
-                int64 = 56, 
-                number = 32.1, 
-                float = 54.3, 
-                double = 67.8, 
-                string = 'a', 
-                byte = 'YQ==', 
-                binary = bytes(b'blah'), 
```

Comment:
Why is the binary assignment gone?

Reply:
If we're removing returning binary, it shouldn't still be there should it?

Reply:
Let's still keep the field, but just give it a string value.
Per [here](https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/python/petstore_api/models/format_test.py#L43)
the bytes variable exists in the model and accepts a string value.
When you do your next PR we will see the type there change to byes, and we can update the sample value for this parameter here.